src/commands/scanner/run.ts

```diff
@@ -114,8 +114,8 @@ export default class Run extends ScannerCommand {
 		// First, we need to do some input validation that's a bit too sophisticated for the out-of-the-box flag validations.
 		this.validateFlags();
 
-		// if severty-for-error flag is used, we want to make sure the severities are normalized
-		const normalizeSeverity: boolean = this.flags['normalize-severity'] || this.flags['severity-threshold'];
+		// if severity-threshold flag is used, we want to make sure the severities are normalized
+		const normalizeSeverity: boolean = (this.flags['normalize-severity'] || this.flags['severity-threshold']) as boolean;
```

Comment:
The linter is incapable of detecting the actual types of the flags, so we're using the `as` keyword to let the compiler know. That's happening liberally throughout this class.

Reply:
@jfeingold35 Could you point me to which rule checks for this?